index.html

```diff
@@ -6,9 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Bootstrap Playground</title>
   </head>
-  <body>
-    <div class="container">
-      Nothing here yet...
+
+
+   
+
+  <div class="accordion accordion-flush" id="accordionExample">
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingOne">
+        <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+          Accordion Item #1
+        </button>
+      </h2>
+      <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne" data-bs-parent="#accordionExample">
+        <div class="accordion-body">
+         <a href="https://github.com/" style="text-decoration: none; color: rgb(70, 51, 0);">garnix</a>
```

Comment:
It's a good practice to define styles within a CSS file, instead of adding them into a style attribute. Could you move this styles there and just add the new class to the link?